src/library_wasmfs.js

```diff
@@ -366,7 +366,14 @@ FS.createPreloadedFile = FS_createPreloadedFile;
         return __wasmfs_rename(oldPathBuffer, newPathBuffer);
       }));
     },
-    // TODO: syncfs
+    syncfs: (populate, callback) => {
+      if (typeof populate == 'function') {
```

Comment:
It looks like this is handling support for two forms of calls. Please add a comment to clarify what they are.

Reply:
I think I kept this section from the old API for compatibility, but now that I think about it, is `wasmfs_flush()` responsible for initializing the filesystem with existing data from the persistent source? Would `populate` even be necessary?

Reply:
I think for now we don't need populate (I'm not sure we have a design for how to do that kind of thing yet, actually), so let's remove it here.